README.md

```
@@ -0,0 +1,42 @@
+<p align="center">
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: p

(MD033, no-inline-html)